### PR TITLE
Merge dev scheduled 2021 dec 10

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
Merges from development

### CODE REVIEW

**Review by raduchis — DISMISSED** [review]

File: process/coordinator/process.go [hunk 1]
```diff
@@ -566,20 +566,15 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
...
 			"header round", hdr.GetRound(),
 			"header nonce", hdr.GetNonce(),
 			"num mini blocks to be processed", len(crossMiniBlockInfos),
-			"num mini blocks processed", nrMiniBlocksProcessed,
-			"total gas provided", tc.gasHandler.TotalGasProvided(),
-			"total gas refunded", tc.gasHandler.TotalGasRefunded(),
-			"total gas penalized", tc.gasHandler.TotalGasPenalized())
-
-		log.Debug("transactionCoordinator.CreateMbsAndProcessCrossShardTransactionsDstMe: gas consumed, refunded and penalized info",
 			"num mini blocks processed", numMiniBlocksProcessed,
```
Comment (mariusmihaic) [inline process/coordinator/process.go:589]: Was this deleted on purpose?

File: consensus/spos/scheduledProcessor.go [hunk 2]
```diff
@@ -209,13 +209,6 @@ func (sp *scheduledProcessorWrapper) setStatus(status processingStatus) {
 	sp.status = status
 }
 
-func (sp *scheduledProcessorWrapper) getStartTime() time.Time {
```
Comment (mariusmihaic) [inline consensus/spos/scheduledProcessor.go:212]: Was this unused?

**Review by mariusmihaic — APPROVED** [review]
Looks good, just a few questions for clarification

File: consensus/spos/scheduledProcessor.go (hunk 2 above)
Comment (AdoAdoAdo) [reply consensus/spos/scheduledProcessor.go:212]: yes removed because it was unused

File: process/coordinator/process.go (hunk 1 above)
Comment (AdoAdoAdo) [reply process/coordinator/process.go:589]: yes the log was duplicated

**Review by raduchis — APPROVED** [review]

**Review by mariusmihaic — APPROVED** [review]

**Pull request merged**